flybywiresim/aircraft
# refactor: move to official navigraph SDK for charts

This fixes some authentication issues with Navigraph by moving to their official SDK.

## Summary of Changes

* Move to official Navigraph SDK
* Fix chart name being inaccurate

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


* Make sure SImBrief import (both with logged-in Navigraph account and override SimBrief ID) still works correctly
* Make sure Charts functionality still works correctly


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
